dist-git/dist-git.spec

```diff
@@ -131,6 +134,10 @@ cp -a configs/httpd/dist-git.conf %{buildroot}%{_sysconfdir}/httpd/conf.d/
 cp -a configs/httpd/dist-git/* %{buildroot}%{_sysconfdir}/httpd/conf.d/dist-git/
 cp -a configs/systemd/*        %{buildroot}%{_unitdir}/
 
+%if 0%{?rhel} && 0%{?rhel} > 9
```

Comment:
|| 0%{?fedora}

Reply:
oh, or drop this condition entirely; it is fine to install the conf file on EPEL9 and older ... they just have no effect there